recipes/mmtf-pyspark/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: mmtfpyspark
+  version: "0.2.0"
+
+source:
+  git_url: https://github.com/sbl-sdsc/mmtf-pyspark
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - pyspark
+    - biopython
+    - msgpack-python
+    - mmtf-python
+    - sympy
+    - requests
+  run:
+    - python
+    - numpy x.x
+    - pyspark
+    - biopython
+    - msgpack-python
+    - mmtf-python
+    - sympy
+    - requests
+
+test:
+  imports:
+    - mmtfPyspark
+
+about:
+  home: https://github.com/sbl-sdsc/mmtf-pyspark
+  license: Apache License 2.0
```

Comment:
`license: Apache-2.0`

Also please add the `license_file`. It's not in the pypi tarball, but you can manually package it:
https://conda-forge.org/docs/meta.html#packaging-the-license-manually

also please add the `dev_url` and `doc_url`:

https://github.com/sbl-sdsc/mmtf-pyspark

and:

http://mmtf-pyspark.readthedocs.io/

respectively.